### PR TITLE
Updates packaging script for HSDK 2.7 using new --add option

### PR DESCRIPTION
**Pull request by mocsharp**
HSDK 2.7 introduces a new `--add` option for the Holoscan Pacakger, which enables adding additional files and dependencies to the HAP container image.  

This PR updates the Endoscopy Tool Tracking packaging scripts to use the new `--add` option.

Please merge after HSDK 2.7 release.

### CODE REVIEW

File: applications/endoscopy_tool_tracking/python/package-app.sh [hunk 1]
```diff
@@ -32,18 +33,15 @@ if [ $(get_host_arch) == "aarch64" ]; then
...
+echo -e "${YELLOW}holoscan package -c $APP_PATH/endoscopy_tool_tracking.yaml --platform [igx-orin-devkit | jetson-agx-orin-devkit | sbsa, x64-workstation] --platform-config [igpu | dgpu] -t holohub-endoscopy-tool-tracking-python --include onnx holoviz --add $INSTALL_DIR/lib/ --add $INSTALL_DIR/python/lib/ $APP_PATH/endoscopy_tool_tracking.py${NOCOLOR}"
 echo -e "\nFor example:"
-echo -e "${YELLOW}holoscan package -c $APP_PATH/endoscopy_tool_tracking.yaml --platform ${PLATFORM} --platform-config ${GPU} -t holohub-endoscopy-tool-tracking-python $APP_PATH/endoscopy_tool_tracking.py --include onnx holoviz${NOCOLOR}"
+echo -e "${YELLOW}holoscan package -c $APP_PATH/endoscopy_tool_tracking.yaml --platform ${PLATFORM} --platform-config ${GPU} -t holohub-endoscopy-tool-tracking-python --include onnx holoviz --add $INSTALL_DIR/lib/ --add $INSTALL_DIR/python/lib/ $APP_PATH/endoscopy_tool_tracking.py${NOCOLOR}"
```
Comment (tbirdso) [inline applications/endoscopy_tool_tracking/python/package-app.sh:44]: Is `--add` recursive, or do we also need `--add $INSTALL_DIR/lib/gxf_extensions/` to match `cp` commands above?

**Review by tbirdso — APPROVED** [review]
Looks great! 💯

File: applications/endoscopy_tool_tracking/python/package-app.sh (hunk 1 above)
Comment (mocsharp) [reply applications/endoscopy_tool_tracking/python/package-app.sh:44]: It is recursive.

**Comment by tbirdso** [comment]
@mocsharp does this need re-review / merge?

**Comment by mocsharp** [comment]
> @mocsharp does this need re-review / merge?

Yes, please merge. I have verified the CPP app is working. The Python will not work until 2.8 with the fix to relax pip version pinning.

**Review by tbirdso — APPROVED** [review]

**Pull request merged**